### ISSUE TITLE
[ENHANCEMENT REQUEST] - Quote attribute values in selectors

### ISSUE THREAD

**Issue by egaudette** [sasstools/sass-lint#707]
This request is for the enforcement of quotation of attributes in selectors, as such:

**Example(s) of enforcement:**
default enforcement:
```yaml
attribute-quotes: 2
```
explicit enforcement:
```yaml
  attribute-quotes:
    - 2
    -
      include: true;
```

The following would be allowed:

```scss
input[type="checkbox"] {
    color: #aaa;
}
```

The following would be disallowed:
```scss
input[type=checkbox] {
    color: #aaa;
}
```

<br/>
<br/>

**Example(s) of non-enforcement:**
default non-enforcement:
```yaml
attribute-quotes: 0
```
explicit non-enforcement:
```yaml
  attribute-quotes:
    - 0
    -
      include: false;
```

The following would be allowed:

```scss
input[type=checkbox] {
    color: #aaa;
}
```

The following would be disallowed:
```scss
input[type="checkbox"] {
    color: #aaa;
}
```

**Comment by bgriffith** [comment]
Thanks for the request :+1: Should be an easy one to implement. Surprised we don't have it covered already!

**Comment by egaudette** [comment]
So was I, honestly. I thought I had missed it the first couple of go arounds. I didn't notice it when I checked over the rules, again. If it is nested in a rule that I missed, I apologize.

**Comment by DanPurdy** [comment]
@egaudette it's definitely missing should be resolved with #708 if you want to check to see it suits your needs, thanks for the suggestion 👍

**Comment by egaudette** [comment]
Sounds good!

Currently I'm using:
"gulp-sass-lint": "1.7.0"

I'm weary of utilizing untested versions, and therefore don't utilize the the wildcard '*' when doing builds:

"gulp-sass-lint": "*"

Is there any sort of email distribution list when there is a new build & distributes of gulp-sass-lint? As those happen, I would like to test that newest version before utilizing it in my Production builds.

**Comment by DanPurdy** [comment]
Hmm sorry, we're not usually that organised, we kind of release when we feel we've done enough 😄 we sometimes announce it on Twitter https://twitter.com/DanPurdy0 but that's all i can think of for now.

I would imagine if you check back by the middle of next week you will see that 1.8 is released.

**Comment by egaudette** [comment]
Righty oh! Will do.

**Comment by egaudette** [comment]
@DanPurdy This is working splendidly. Thanks!
